RecoParticleFlow/PFClusterProducer/python/particleFlowBadHcalPseudoCluster_cff.py

```diff
@@ -0,0 +1,10 @@
+from RecoParticleFlow.PFClusterProducer.particleFlowBadHcalPseudoCluster_cfi import *
+
+# OFF by default, turned on via modifier for the moment
+particleFlowBadHcalPseudoCluster.enable = cms.bool(False)
+
+from Configuration.Eras.Modifier_run2_HE_2018_cff import run2_HE_2018
+run2_HE_2018.toModify(particleFlowBadHcalPseudoCluster, thresholdHE = 4)
+
+from Configuration.Eras.Modifier_run3_HB_cff import run3_HB
+run3_HB.toModify(particleFlowBadHcalPseudoCluster, thresholdHB = 4, thresholdHE = 4)
```

Comment:
Honestly I don't know who many depths we will have in Phase1 HB. 
For Phase 1 HE, 4 looked ok in some debugging.

Reply:
Phase 1 HB has 4 depths. Phase 1 HE has up to 7 depths.

Also, please avoid setting HE parameters in the `run3_HB` modifier. The setting from `run2_HE_2018` is sufficient.

Reply:
In fact, the topology has various `maxDepth` functions, so it would be preferable to pick up the values automatically.
https://github.com/cms-sw/cmssw/blob/9df145f7daff289c67228913a00d2e8e1a1109e6/Geometry/CaloTopology/interface/HcalTopology.h#L145-L147

Reply:
ok, from this and the discussion above I think it may just be simpler to code the logic as "consider bad an (ieta,iphi) region if at least half of the dephts are bad", which would also avoid the need of configuration changes across eras.